src/main/java/teammates/storage/api/FeedbackResponsesDb.java

```diff
@@ -532,7 +534,9 @@ public FeedbackResponse getFeedbackResponseEntityOptimized (FeedbackResponseAttr
                 new ArrayList<FeedbackResponseAttributes>();
         
         for (FeedbackResponse fr : frList) {
+            if (!JDOHelper.isDeleted(fr)) {
```

Comment:
Should we have some comments to clarify why we need to check if object is deleted for these 2 functions while other retrieval functions do not have ?